Week-11/theme-app/src/App.js

```diff
@@ -0,0 +1,47 @@
+import './App.css';
+import { BsMoonStarsFill, BsFillSunFill } from 'react-icons/bs';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { handleDarkMode } from './store/actions/darkModeAction';
+
+function App() {
+	const dispatch = useDispatch();
+
+	const mode = useSelector((state) => state.darkMode);
+
+	const { isDarkMode } = mode;
+
+	const switchDarkMode = () => {
+		isDarkMode
+			? dispatch(handleDarkMode(false))
+			: dispatch(handleDarkMode(true));
+	};
+
+	useEffect(() => {
+		//changing color of body with darkMode in useEffect
+		document.body.style.backgroundColor = isDarkMode ? '#292c35' : '#fff';
+	}, [isDarkMode]);
+
```

Comment:
better way is to handle through CSS by toggling the class name for the APp container